src/data/markdown/docs/01 guides/01 Getting started/03 Running k6.md

```diff
@@ -221,63 +181,26 @@ export default function() {
 
 </div>
 
-## Using checks
-
-Maybe we want to verify that an HTTP transaction worked and that the transaction time stayed below some acceptable value. We can use the [check()](/javascript-api/k6/check-val-sets-tags) function for this:
-
-<div class="code-group" data-props='{"labels": ["stages.js"], "lineNumbers": [true]}'>
+## Running cloud tests
 
-```javascript
-import http from 'k6/http';
-import { check, sleep } from 'k6';
-
-export let options = {
-  vus: 10,
-  duration: '30s',
-};
-
-export default function() {
-  let res = http.get('http://test.k6.io');
-  check(res, {
-    'status was 200': r => r.status == 200,
-    'transaction time OK': r => r.timings.duration < 200,
-  });
-  sleep(1);
-}
-```
+One of the goals with k6 is to support three execution modes to run your k6 tests:
 
-</div>
+- [Local](#running-local-tests): on your local machine or a CI server.
+- [Cloud](/cloud): on cloud infrastructure managed by `k6 Cloud`.
```

Comment:
Why is k6 Cloud in a back-tick "code" styling?

Reply:
Initially, it was a link. Then, I created the link to the beginning of the phrase. 

I didn't want to have two same links on the same sentence and removed that one and used `backtick`.

Reply:
https://github.com/loadimpact/k6-docs/pull/26/commits/63afdcc5f86e883180b9cb050c595edea88c4720